app/controllers/admin/users/tools/credits_controller.rb

```diff
@@ -0,0 +1,87 @@
+module Admin
+  module Users
+    module Tools
+      class CreditsController < Admin::ApplicationController
+        layout false
+
+        def show
+          user = ::User.find(params[:user_id])
+
+          render CreditsComponent.new(user: user), content_type: "text/html"
```

Comment:
This seems like it would be easy to forget, we may want to eventually wrap this up in a helper in `Admin::ApplicationController`, something like this (completely untested):

```ruby
def render_component(component_class, *args, **kwargs)
  render component_class.new(*args, **kwargs), content_type: "text/html"
end
```

Then we call it like this:

```ruby
render_component CreditsComponent, user: user
```

The above feels quite Rails-y to me and eliminates a whole possible error class.